runtime/oti/j9.h

```diff
@@ -375,6 +375,10 @@ static const struct { \
 #define J9_SHARED_CACHE_DEFAULT_BOOT_SHARING(vm) FALSE
 #endif /* defined(OPENJ9_BUILD) */
 
+#if defined(J9VM_OPT_SHR_MSYNC_SUPPORT)
+#define	J9SHR_MSYNC_SUPPORT TRUE
```

Comment:
> Please use a space instead of a tab after define.

Changed it to space.

> But why do we need/want this at all? We can use J9VM_OPT_SHR_MSYNC_SUPPORT directly.

We can replace the old macro `J9SHR_MSYNC_SUPPORT` with `J9VM_OPT_SHR_MSYNC_SUPPORT`. I don't want to make this PR bigger, so I didn't do that in this PR. It can be done in a separate PR (or separate commit), which could make it easier for reviewers to review.

Reply:
This changes 7 out of the 8 files that used `J9SHR_MSYNC_SUPPORT` before; changing the 8th file instead of this makes the change smaller.

Reply:
Removed `J9SHR_MSYNC_SUPPORT` and use `J9VM_OPT_SHR_MSYNC_SUPPORT` directly.